code/modules/reagents/reagent_containers/drinks/bottle.dm

```diff
@@ -214,8 +214,8 @@
 			sleep(sleep_not_stacking) //Not stacking
 			stop_spin_bottle = FALSE
 
-/obj/item/chems/drinks/bottle/pickup(mob/living/user)
-	animate(src, transform = null, time = 0) //Restore bottle to its original position
+/obj/item/chems/drinks/bottle/on_picked_up(mob/living/user)
+	transform = null //Restore bottle to its original position
```

Comment:
I'm pretty sure this was necessary to interrupt `SpinAnimation`. Without using `animate` I think the bottle spin animation will continue in your inventory even after you pick it up. I've yet to test it locally though, so I could be wrong.
```suggestion
	animate(src, transform = null, time = 0) //Restore bottle to its original position and interrupt spin animation
```